tests/Load/config.sh

```
@@ -0,0 +1,67 @@
+ROLE_NAME=${ROLE_NAME:-$DEFAULT_ROLE_NAME}
+BRANCH_NAME=${BRANCH_NAME:-$DEFAULT_BRANCH_NAME}
+SECURITY_GROUP_NAME=${SECURITY_GROUP_NAME:-$DEFAULT_SECURITY_GROUP_NAME}
+LOCAL_MODE=${LOCAL_MODE:-$DEFAULT_LOCAL_MODE}
+
+if [[ "$LOCAL_MODE" == "true" ]]; then
+    export ENDPOINT_URL=$AWS_SQS_ENDPOINT_BASE:$LOCALSTACK_PORT
+    export AWS_ACCESS_KEY_ID=$AWS_SQS_KEY
+    export AWS_SECRET_ACCESS_KEY=$AWS_SQS_SECRET
+    AWS_CLI="aws --endpoint-url=${ENDPOINT_URL}"
+else
+    AWS_CLI="aws"
```

Shellcheck

[warning] AWS_CLI appears unused. Verify use (or export if used externally).

(SC2034)